doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,148 @@
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.07` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.08` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) workloads. |
+| `req.inf.ntw.09` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.10` | Network | The networking solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.11` | Network | The networking solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.12` | Network | The Architecture **must** support dual stack IPv4 and IPv6 for Kubernetes workloads. |
+| `req.inf.ntw.13` | Network | The Architecture **should** support IPv6 for Kubernetes workloads. |
+| `req.inf.ntw.14` | Network | The Architecture **must** support Conatiner Network Interface (CNI). |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+| `req.inf.vir.01`   | Virtualisation |   The Architecture **must** support the capability for Containers to consume virtualised compute, storage and network resources.|
+| `req.inf.phy.01`  |  Physical Infrastructure |   The Architecture **must** support the capability for Containers to consume physical (aka bare metal) compute, storage and network resources. |
+
+<p align="center"><b>Table 2-2:</b> Kubernetes Architecture: Infrastructure Requirements</p>
+Please note that "shared" is a reference to multi-tenant support and pooled storage resources.
+
+<a name="2.3.3"></a>
+### 2.3.3 Kubernetes Cluster Requirements
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.kcm.01` | General | The Architecture **must** allow infrastructure resource sharing within a Kubernetes cluster. |
+| `req.kcm.02` | General | The Architecture **must** allow discovery and management of NFVI resources. |
+| `req.kcm.03` | General | The Architecture **must** support Enhanced Platform Awareness (EPA). |
```

Comment:
AFAIK, kubernetes doesn't provides scheduling based on EPA features. Shall this also be mentioned as a **must** or **should** requirement ?
```The Architecture **must** support scheduling of workloads based on EPA features``` or something like ```The Architecture **must** support scheduling and discoverability of workloads based on EPA features```
Pls correct me if I am wrong.

Reply:
@msidana modified to the following -- please critique:
| `req.kcm.02` | General | The Architecture **must** support discoverability of nodes and their features. |
| `req.kcm.03` | General | The Architecture **must** support scheduling of workloads based on Enhanced Platform Awareness (EPA) features such as CPU Pinning, huge-pages and SR-IOV. |

Reply:
Looks good.